validation-test/stdlib/ArrayTrapsObjC.swift

```diff
@@ -24,11 +18,11 @@ class Derived2 : Derived { }
 ArrayTraps.test("downcast1")
   .skip(.custom(
     { _isFastAssertConfiguration() },
-    reason: "this trap is not guaranteed to happen in -Ounchecked"))
+    reason: "this trap is not guaranteed to happen in -Unchecked"))
```

Comment:
is this right?

Reply:
My working theory is I fat-fingered a spellcheck keypresss...